consensus/scp/src/node/mod.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+//! A node of the consensus network.
+
+mod node_impl;
+mod node_trait;
+
+pub use node_impl::*;
+pub use node_trait::*;
```

Comment:
Unless it's causing issues, I'd recommend this explicitly uses the elements you need, rather than this, because this will also export the results of use statements in these nodes, AFAIK.